packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetails.tsx

```diff
@@ -33,7 +36,7 @@ type ModelVersionsDetailProps = {
   'breadcrumb' | 'title' | 'description' | 'loadError' | 'loaded' | 'provideChildrenPadding'
 >;
 
-const ModelVersionsDetails: React.FC<ModelVersionsDetailProps> = ({ tab, ...pageProps }) => {
+const ModelVersionsDetailsContent: React.FC<ModelVersionsDetailProps> = ({ tab, ...pageProps }) => {
```

Comment:
I was about to suggest we avoid renaming this and instead change the default export to something like ModelVersionsDetailsWrapper, but this is the same impact so it's fine